src/notifications/common.ts

```diff
@@ -51,10 +51,16 @@ export const notificationPreferenceMap: Partial<
 > = {
   [NotificationType.ArticleNewComment]: NotificationPreferenceType.Post,
   [NotificationType.CommentReply]: NotificationPreferenceType.Comment,
+  [NotificationType.SquadReply]: NotificationPreferenceType.Comment,
   [NotificationType.SquadPostAdded]: NotificationPreferenceType.Source,
   [NotificationType.SquadMemberJoined]: NotificationPreferenceType.Source,
 };
 
+export const commentNotificationTypes = [
```

Comment:
We can use `notificationPreferenceMap` to generate a reverse map. From `NotificationPreferenceType` to `NotificationType[]`

Reply:
There could be of `NotificationPreferenceType` but not applicable to the case. Let's say we have another `NotificationType` relevant to comments, it should not be included in the use case here. Probably I should rename this to be more explicit that this is about replies.